torch/csrc/distributed/c10d/init.cpp

```diff
@@ -534,6 +535,21 @@ They are used in specifying strategies for reduction collectives, e.g.,
       py::arg("bucket_size"),
       py::call_guard<py::gil_scoped_release>());
 
+  module.def(
+      "_broadcast_coalesced",
```

Comment:
Do you plan to eventually replace the existing `_dist_broadcast_coalesced` with this one? IIUC, this is still distributed broadcast. Any reason for not including a `dist` in the name?

Reply:
I think we can remove `_dist_broadcast_coalesced` once this lands. I don't like having `dist` in the name, since it is implied by the namespace. When we do work on the C++ API a bit more, we'll have the CUDA version as `torch::cuda::broadcast_coalesced` and the distributed version as `torch::distributed::broadcast_coalesced`. No need to repeat `dist` in the name IMO.